andela/wg-ravens
# #157589652 Import and Export workouts

**What does this PR do?**
Adds import and export functionality for workouts

**Description of Task to be completed?**
The export feature allows the user to send their workouts to other users so they can add them in their workout schedules. The import feature ensures that one can upload a workout sent to him/her.

**How should this be manually tested?**
While on the app, click on `Workouts` then `workouts` on the drop-down menu. To your right, below add workout you will see a place where you can import workouts in .csv format. Add a .csv file and see it appearing on the workout list.

You can also export a workout by clicking on the workout, then options on the drop-down menu. You will see the export workout option. After clicking it the workout will be downloaded as a .csv file format.

**What are the relevant pivotal tracker stories?**
#157589652

**Screenshots**

<img width="1277" alt="screen shot 2018-06-18 at 00 10 15" src="https://user-images.githubusercontent.com/16057093/41512102-37202df6-728c-11e8-94fd-a5e597bcc562.png">

<img width="1280" alt="screen shot 2018-06-18 at 00 10 44" src="https://user-images.githubusercontent.com/16057093/41512107-48ac564e-728c-11e8-8e38-04a26974e656.png">

_Sample exported workout_

<img width="638" alt="screen shot 2018-06-18 at 00 11 11" src="https://user-images.githubusercontent.com/16057093/41512112-52554250-728c-11e8-85ae-5b7bc6be7382.png">
